[0m15:03:24.880869 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd077259160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd078488e10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd0763e3ed0>]}


============================== 15:03:24.893711 | 30e3faec-2da4-4dce-840f-de3d1456fb56 ==============================
[0m15:03:24.893711 [info ] [MainThread]: Running with dbt=1.9.1
[0m15:03:24.897368 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init dbtYalo', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:03:25.012684 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.13/site-packages/dbt/include/starter_project
[0m15:03:25.379497 [info ] [MainThread]: 
Your new dbt project "dbtYalo" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:03:25.380426 [info ] [MainThread]: Setting up your profile.
[0m15:03:43.190068 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 18.442362, "process_in_blocks": "2584", "process_kernel_time": 0.634687, "process_mem_max_rss": "90532", "process_out_blocks": "1696", "process_user_time": 1.962831}
[0m15:03:43.191620 [debug] [MainThread]: Command `dbt init` succeeded at 15:03:43.191481 after 18.45 seconds
[0m15:03:43.192229 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd076409e00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd076273e30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fd076472140>]}
[0m15:03:43.192943 [debug] [MainThread]: Flushing usage events
[0m15:03:48.974600 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:37:47.161128 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c6f72c590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c70aaca50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c6eece490>]}


============================== 15:37:47.168470 | c01a7604-1413-4e94-b993-e047baa70549 ==============================
[0m15:37:47.168470 [info ] [MainThread]: Running with dbt=1.9.1
[0m15:37:47.169521 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m15:37:47.186465 [info ] [MainThread]: dbt version: 1.9.1
[0m15:37:47.187104 [info ] [MainThread]: python version: 3.13.1
[0m15:37:47.187649 [info ] [MainThread]: python path: /usr/local/bin/python
[0m15:37:47.188138 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[0m15:37:47.196958 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m15:37:47.197549 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m15:37:47.198066 [info ] [MainThread]: Using dbt_project.yml file at /code/dbt_project.yml
[0m15:37:47.198701 [info ] [MainThread]: Configuration:
[0m15:37:47.199127 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m15:37:47.199551 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m15:37:47.199944 [info ] [MainThread]: Required dependencies:
[0m15:37:47.200347 [debug] [MainThread]: Executing "git --help"
[0m15:37:47.202695 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m15:37:47.203182 [debug] [MainThread]: STDERR: "b''"
[0m15:37:47.203570 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m15:37:47.203993 [info ] [MainThread]: Connection test skipped since no profile was found
[0m15:37:47.204738 [info ] [MainThread]: [31m2 checks failed:[0m
[0m15:37:47.205596 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Compilation Error
    Could not render {{ env('DBT_PROJECT_DIR') }}/keys/yalo-ae-case-cdf7c467547d.json: 'env' is undefined


[0m15:37:47.206407 [info ] [MainThread]: Project loading failed for the following reason:
 project path </code/dbt_project.yml> not found

[0m15:37:47.208204 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.10886751, "process_in_blocks": "75152", "process_kernel_time": 0.317217, "process_mem_max_rss": "89188", "process_out_blocks": "0", "process_user_time": 1.095843}
[0m15:37:47.208789 [debug] [MainThread]: Command `dbt debug` failed at 15:37:47.208688 after 0.11 seconds
[0m15:37:47.209255 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c6e9e6c40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c6ea3fe30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1c6ea31590>]}
[0m15:37:47.209818 [debug] [MainThread]: Flushing usage events
[0m15:37:47.989528 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:50:26.650305 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4560ced160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4562128e10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f456001fed0>]}


============================== 15:50:26.658373 | 7690c28a-2074-417d-ab37-19ee70c1ecfd ==============================
[0m15:50:26.658373 [info ] [MainThread]: Running with dbt=1.9.1
[0m15:50:26.659086 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:50:26.670384 [info ] [MainThread]: dbt version: 1.9.1
[0m15:50:26.671038 [info ] [MainThread]: python version: 3.13.1
[0m15:50:26.671531 [info ] [MainThread]: python path: /usr/local/bin/python
[0m15:50:26.671984 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[0m15:50:26.681173 [error] [MainThread]: Encountered an error:
Parsing Error
  Env var required but not provided: 'DBT_PROJECT_DIR'
[0m15:50:26.682224 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.09333466, "process_in_blocks": "11504", "process_kernel_time": 0.229431, "process_mem_max_rss": "90600", "process_out_blocks": "1696", "process_user_time": 1.107256}
[0m15:50:26.682899 [debug] [MainThread]: Command `dbt debug` failed at 15:50:26.682786 after 0.09 seconds
[0m15:50:26.683352 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4560046fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f45600d1250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4560089bf0>]}
[0m15:50:26.683830 [debug] [MainThread]: Flushing usage events
[0m15:50:27.454031 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:50:50.241932 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc6b78590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc7f0ca50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc6302490>]}


============================== 15:50:50.245063 | fd484653-2a1a-4b30-a267-a9cd8ac5b880 ==============================
[0m15:50:50.245063 [info ] [MainThread]: Running with dbt=1.9.1
[0m15:50:50.245743 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/root/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m15:50:50.252745 [info ] [MainThread]: dbt version: 1.9.1
[0m15:50:50.253286 [info ] [MainThread]: python version: 3.13.1
[0m15:50:50.253742 [info ] [MainThread]: python path: /usr/local/bin/python
[0m15:50:50.254162 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[0m15:50:50.262725 [info ] [MainThread]: Using profiles dir at /root/.dbt
[0m15:50:50.263302 [info ] [MainThread]: Using profiles.yml file at /root/.dbt/profiles.yml
[0m15:50:50.263778 [info ] [MainThread]: Using dbt_project.yml file at /code/dbt_project.yml
[0m15:50:50.264345 [info ] [MainThread]: Configuration:
[0m15:50:50.264739 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m15:50:50.265166 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m15:50:50.265634 [info ] [MainThread]: Required dependencies:
[0m15:50:50.266194 [debug] [MainThread]: Executing "git --help"
[0m15:50:50.269358 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m15:50:50.269901 [debug] [MainThread]: STDERR: "b''"
[0m15:50:50.270289 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m15:50:50.270753 [info ] [MainThread]: Connection test skipped since no profile was found
[0m15:50:50.271366 [info ] [MainThread]: [31m2 checks failed:[0m
[0m15:50:50.271791 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Compilation Error
    Could not render {{ env('DBT_PROJECT_DIR') }}/key/yalo-ae-case-cdf7c467547d.json: 'env' is undefined


[0m15:50:50.272567 [info ] [MainThread]: Project loading failed for the following reason:
 project path </code/dbt_project.yml> not found

[0m15:50:50.274146 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.08618716, "process_in_blocks": "0", "process_kernel_time": 0.110385, "process_mem_max_rss": "89476", "process_out_blocks": "0", "process_user_time": 0.983431}
[0m15:50:50.274769 [debug] [MainThread]: Command `dbt debug` failed at 15:50:50.274655 after 0.09 seconds
[0m15:50:50.275328 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc5e028b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc5e5fe30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcfc5e51590>]}
[0m15:50:50.275916 [debug] [MainThread]: Flushing usage events
[0m15:50:50.954222 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:53:32.029386 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4e78cc20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4fbb0910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4dfd25d0>]}


============================== 21:53:32.035344 | 606cad2e-ac8f-42ce-a46d-4da8257d637a ==============================
[0m21:53:32.035344 [info ] [MainThread]: Running with dbt=1.9.1
[0m21:53:32.035968 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:53:32.036698 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /code/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:53:32.038799 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.0648544, "process_in_blocks": "75728", "process_kernel_time": 0.292933, "process_mem_max_rss": "89544", "process_out_blocks": "1128", "process_user_time": 1.190647}
[0m21:53:32.039849 [debug] [MainThread]: Command `dbt run` failed at 21:53:32.039727 after 0.07 seconds
[0m21:53:32.040371 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4dc0b490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4dd43e30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5d4dd567a0>]}
[0m21:53:32.040828 [debug] [MainThread]: Flushing usage events
[0m21:53:32.798180 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m00:32:15.561310 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b40a50590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b41de4a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b401da490>]}


============================== 00:32:15.565107 | 3e7782e4-191a-4233-bcb0-e7f3ff43d7f7 ==============================
[0m00:32:15.565107 [info ] [MainThread]: Running with dbt=1.9.2
[0m00:32:15.565869 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt deps', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m00:32:15.566855 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /code/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m00:32:15.567974 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_success": false, "command_wall_clock_time": 0.06872211, "process_in_blocks": "112", "process_kernel_time": 0.100167, "process_mem_max_rss": "88952", "process_out_blocks": "0", "process_user_time": 0.981639}
[0m00:32:15.568567 [debug] [MainThread]: Command `dbt deps` failed at 00:32:15.568443 after 0.07 seconds
[0m00:32:15.569181 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b3fcf8510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b3ff27410>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2b3ff36580>]}
[0m00:32:15.569832 [debug] [MainThread]: Flushing usage events
[0m00:32:16.281397 [debug] [MainThread]: An error was encountered while trying to flush usage events
